src/dbnode/client/fetch_tagged_results_accumulator_merge_test.go

```diff
@@ -314,11 +315,24 @@ func (tm testFetchTaggedWorkflow) run() fetchTaggedResultAccumulator {
 	accum.Clear()
 	accum.Reset(tm.startTime, tm.endTime, tm.topoMap, majority, tm.level)
 	for _, s := range tm.steps {
-		opts := fetchTaggedResultAccumulatorOpts{
-			host:     host(tm.t, tm.topoMap, s.hostname),
-			response: s.response,
+		var (
+			done bool
+			err  error
+		)
+		if s.fetchTaggedResult != nil {
+			opts := fetchTaggedResultAccumulatorOpts{
+				host:     host(tm.t, tm.topoMap, s.hostname),
+				response: s.fetchTaggedResult,
+			}
+			done, err = accum.AddFetchTaggedResponse(opts, s.err)
+		} else {
```

Comment:
Instead of if/else can we do a switch and then have a `default` for where they are both nil and do a `require.FailNow(t, ...)`?

Reply:
sure thing

Reply:
done